src/toast.tsx

```diff
@@ -107,7 +107,7 @@ const Toast: FC<ToastProps> = ({
       }
     }
   }
-
+  // @ts-ignore
```

Comment:
If you remove this, or don't type it properly you'll continue getting ts errors. I've already forked this library and applied the fixed so I'm not fussed, but it will likely limit new users to this lib.

I think your core problem is the fact that your type files are sourced via a `.ts` file rather than a `.d.ts`

Reply:
I just published a new version (v2.1.1) compiled to `javascript` with `.d.ts` files. please confirm if this fixes your  typescript issues.

Reply:
All fixed now 😄  I'll close this PR now, cheers mate 🤙